lib/cachex/services/courier.ex

```diff
@@ -91,13 +97,7 @@ defmodule Cachex.Services.Courier do
   #
   # This will update all processes waiting for the result of the
   # specified task, and remove the task from the tracked state.
-  def handle_info({ :notify, key, result }, { cache, tasks }) do
-    normalized = normalize_commit(result)
-
-    with { :commit, val } <- normalized do
-      Put.execute(cache, key, val, const(:notify_false))
-    end
-
+  def handle_info({ :notify, key, normalized }, { cache, tasks }) do
```

Comment:
Can you please change `normalized` back to `result`?

Reply:
I changed what I pass in to `handle_info`; should I call `normalize_commit(result)` again there?

Reply:
No, it's fine - just the name should be `result` here because there's no context as to what `normalized` means anymore :)